projects/demo/src/components/textfield/index.html

```diff
@@ -0,0 +1,32 @@
+<tr>
+    <td colspan="3">
+        <span tuiTitle>
+            <a
+                tuiLink
+                [routerLink]="routes.Textfield"
+            >
+                <strong>Textfield</strong>
```

Comment:
Let's name it `TextfieldOptions` and I'm not sure we need a link to textfield here, it's just 2 options, I don't know if people need to go investigate what bigger picture are they a part of :)

Reply:
Yes, renamed and removed link